src/main/java/mate/academy/dao/MovieSessionDao.java

```diff
@@ -0,0 +1,14 @@
+package mate.academy.dao;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.model.MovieSession;
+
+public interface MovieSessionDao {
+    MovieSession add(MovieSession movieSession);
+
+    Optional<MovieSession> get(Long id);
```

Comment:
```suggestion
    MovieSession get(Long id);
```
create MovieSessionDaoImpl and MovieSessionServiceImpl. The service layer should have the following methods:
```
    MovieSession add(MovieSession movieSession);
    MovieSession get(Long id);
    List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
```

Reply:
But in DAO layer I should return Optional and in service I return Entity(Common mistakes)

Reply:
agree